pages/linux/scrun.md

```diff
@@ -0,0 +1,24 @@
+# scrun
+
+> An OCI (Open Container Initiative) runtime proxy for Slurm.
+> More information: <https://slurm.schedmd.com/scrun.html>.
+
+- Create a container in the current directory:
+
+`scrun create {{container_id}}`
```

Comment:
What do yo  mean by create a container here? Are we creating it from nothing? Where does the image come from? Is this like `docker build`, or does it download the container from somewhere?

It would be nice if this example was specific about where the container comes from.

Reply:
Looks like I misunderstood `scrun`. It seems like scrun is not meant as an interface to users (https://slurm.schedmd.com/SLUG23/containers-slug23.pdf, slide 14), but for the developers of e.g. docker, podman etc (?). 

For me, this raises the question on whether to include `scrun` in tldr at all.

Reply:
Hmmm, interesting.

I don't see why it shouldn't be included, even if it isn't user-facing @gutjuri. But I would suggest it isn't a priority on including it vs eg. `sbatch` for example.

Your call.

Reply:
I'll remove the page then and hope someone with experience using this command will create it.